ruby/basic_ruby/nested_collections.md

```diff
@@ -322,9 +322,9 @@ Amazing! We have found a great solution to returning an array that only contains
 ### Assignment
 <div class="lesson-content__panel" markdown="1">
 
-1. To learn more about using a hash with a nested array, read [this article](https://web.archive.org/web/20220525215038/https://learn.co/lessons/nested-hash-iteration) from Learn.co.
-2. To learn more about using an array with a nested hash, read [the answer to this post](https://stackoverflow.com/questions/50529389/manipulating-output-from-an-array-of-nested-hashes-in-ruby) on Stack Overflow.
-3. Complete the [nested collections](https://github.com/TheOdinProject/ruby-exercises/tree/main/ruby_basics) exercises from the [ruby-exercises repo](https://github.com/TheOdinProject/ruby-exercises) that you previously cloned.
+1. Read more about [using a hash with a nested array](https://web.archive.org/web/20220525215038/https://learn.co/lessons/nested-hash-iteration).
+2. This Stack Overflow answer explains more about [using arrays with a nested hash](https://stackoverflow.com/questions/50529389/manipulating-output-from-an-array-of-nested-hashes-in-ruby).
+3. Complete the nested collections exercises
```

Comment:
The text to replace in the spreadsheet was only the first bit of the whole sentence, not the whole sentence itself - easily missed!
```suggestion
3. Complete the nested collections exercises from the [ruby-exercises repo](https://github.com/TheOdinProject/ruby-exercises) that you previously cloned.
```

Reply:
Noted! I have updated the text and I fixed the lining issues too